pkg/build/api/types.go

```diff
@@ -63,6 +63,31 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildCreatedEvent is the reason associated with the event registered when a build is created.
+	BuildCreatedEventReason = "BuildCreated"
```

Comment:
not sure what the "new status input" is, but the idea was for these to feed toaster notifications, i believe.